app/tests/integration/nimbus/conftest.py

```diff
@@ -13,16 +12,38 @@
     BaseExperimentDataClass,
     BaseExperimentMetricsDataClass,
 )
-from nimbus.remote_settings.pages.dashboard import Dashboard
-from nimbus.remote_settings.pages.login import Login
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
-from selenium.common.exceptions import NoSuchElementException
 
-
-@pytest.fixture
-def ds_issue_host():
-    return os.environ["DS_ISSUE_HOST"]
+APPLICATION_FEATURES = {
+    BaseExperimentApplications.DESKTOP: "No Feature Firefox Desktop",
+    BaseExperimentApplications.FENIX: "No Feature Fenix",
+    BaseExperimentApplications.IOS: "No Feature iOS",
+    BaseExperimentApplications.KLAR: "No Feature Klar for Android",
+    BaseExperimentApplications.FOCUS: "No Feature Focus for Android",
+}
+
+APPLICATION_KINTO_REVIEW_PATH = {
+    BaseExperimentApplications.DESKTOP: (
+        "#/buckets/main-workspace/collections/nimbus-desktop-experiments/simple-review"
+    ),
+    BaseExperimentApplications.FENIX: (
+        "#/buckets/main-workspace/collections/nimbus-mobile-experiments/simple-review"
+    ),
+    BaseExperimentApplications.IOS: (
+        "#/buckets/main-workspace/collections/nimbus-mobile-experiments/simple-review"
+    ),
+    BaseExperimentApplications.KLAR: (
+        "#/buckets/main-workspace/collections/nimbus-mobile-experiments/simple-review"
+    ),
+    BaseExperimentApplications.FOCUS: (
+        "#/buckets/main-workspace/collections/nimbus-mobile-experiments/simple-review"
+    ),
+}
+
+
+def slugify(input):
+    return input.lower().replace(" ", "-").replace("[", "").replace("]", "")
```

Comment:
love this

Reply:
Nice to have this DRY. Should we reuse Django’s slugify function vs rolling a second one w/ potentially slightly different rules?

Reply:
We totally should but unfortunately django's not in this container :( We could lift the source of it and just copy it over, which I thought about, but I wanted to (try XD) to change as little as I could so I just left it with the one we have now.  We can do that as a followup though.